test/functional/page_objects/visualize_page.js

```diff
@@ -776,6 +776,13 @@ export function VisualizePageProvider({ getService, getPageObjects }) {
       .getAttribute('height');
     }
 
+    getDataTableHeaders() {
+      return remote
+      .setFindTimeout(defaultFindTimeout * 2)
+      .findByCssSelector('table.table.table-condensed thead')
+      .getVisibleText();
+    }
```

Comment:
Not sure if I can. Looks like `find` is not defined

`tryForTime failure: find is not defined`